File: GARI_Umbria_2026-08-03/DotNet/AgronicaCoreStd/AgronicaCoreModelsSTD/attivita/centri_di_costo/Macchina.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace AgronicaCoreModelsSTD.attivita.centri_di_costo
{
    public class Macchina : CentroDiCosto
    {
        public Macchina()
        {
            classType = costanti.ClassType.Macchina;
            tipo = Tipo.Macchina;
        }

    }
}

File: GARI_Umbria_2026-08-03/GiasNG/src/app/Model/attivita/centri_di_costo/Macchina.ts
import {CentroDiCosto, Tipo} from './CentroDiCosto';

export class Macchina extends CentroDiCosto {


    constructor() {
        super();
        this.classType = 'Macchina';
        this.tipo = Tipo.Macchina;
    }
}
